posthog/warehouse/data_load/service.py

```diff
@@ -88,45 +85,31 @@ def to_temporal_schedule(
         ),
     )
 
-    # Determine spec based on frequency
-    if sync_frequency <= timedelta(hours=1):
-        spec = ScheduleSpec(intervals=[ScheduleIntervalSpec(every=sync_frequency)], jitter=jitter)
-    else:
-        spec = ScheduleSpec(
-            calendars=[get_calendar_spec(hour_of_day, minute_of_hour, sync_frequency)],
-            jitter=jitter if minute_of_hour == 0 and hour_of_day == 0 else None,
-        )
+    sync_time = time(hour_of_day, minute_of_hour)
+    schedule_start = datetime.combine(datetime.today(), sync_time, tzinfo=UTC)
+
+    # Create the spec for the schedule based on the sync frequency and sync time
+    # The sync time is applied using a combination of the offset and the start_at time
+    spec = ScheduleSpec(
+        intervals=[
+            ScheduleIntervalSpec(
+                every=sync_frequency,
+                offset=timedelta(minutes=sync_time.hour * 60 + sync_time.minute) % sync_frequency,
+            )
+        ],
+        start_at=schedule_start,
+        # if custom sync time, don't apply jitter
+        jitter=jitter if minute_of_hour == 0 and hour_of_day == 0 else None,
```

Comment:
keeping existing behaviour here but not sure if we want to apply a small jitter even in this case?

Reply:
Yeah, I think not having jitter is good